src/getHumanAge.test.js

```diff
@@ -7,4 +7,25 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should convert cat and dog ages correctly', () => {
+    expect(getHumanAge(27, 27)).toEqual([2, 2]);
+  });
+
+  test('should return 0 for ages less than the first threshould', () => {
+    expect(getHumanAge(10, 10)).toEqual([0, 0]);
+  });
+
+  test('should return 1 for ages between the first and second threshould',
+    () => {
+      expect(getHumanAge(17, 17)).toEqual([1, 1]);
+    });
+
+  test('should handle large age values correctly', () => {
+    expect(getHumanAge(50, 50)).toEqual([8, 7]);
```

Comment:
The test case for handling large age values is incorrect. According to the task description, for 50 cat years, the expected human age should be 11, and for 50 dog years, it should be 9. Update the expected result to `[11, 9]`.